docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
         target: /wallet.json
         read_only: true
       
-      - /bundles:/bundles
+      - bundles:/bundles
```

Comment:
While the original form is clearly wrong, I'm not sure if we want to have bundles stored in a volume. I think the original intention is to store them under `./bundles`.

@JesseTheRobot @joshbenaron can you confirm which on is preferred: separate volume like introduced by this change, or `./bundles` host side folder?

Here is the link to the original change introducing this: https://github.com/Bundlr-Network/validator/pull/42/commits/f2e253a45c65da6369cd0fea648a43c1290c161d